monkestation/code/game/objects/items/storage/boxes.dm

```diff
@@ -0,0 +1,18 @@
+/*
+ *	Pre-packed boxes for testing purposes
+ */
+
+/obj/item/storage/box/oilgrenade
```

Comment:
This file doesn't appear to be in the .dme
May want to remove it if it's no longer needed for testing.